### DISCUSSION TITLE
Onedrive help

### DISCUSSION THREAD

**Discussion by Saorsa32** [remotely-save/remotely-save#135] [General]
Hi I followed all the instructions and see the folder on my Android phone, but when I am in obsidian and try to open a new vault I can't find the app/onedrive folder. I have a Galaxy S10. Where do I need to look to find it?

**Comment by fyears** [comment]
The instruction means that your vault files would be uploaded to onedrive's `apps/remotely-save` folder (and you can preview the uploaded files using onedrive's web page), but it doesn't mean that it would create a local folder `apps/...`. You local folder location is not changed.

**Reply by Saorsa32** [reply]
So, if I want to open the remotely saved file on my obsidian app, how do I do that? Usually to open vault I go to open folder as vault, but I can't find my my vaults.

**Reply by sytone** [reply]
Make a new vault with the same name and then install the plugin to that vault. Configure the settings to connect to OneDrive and then it will sync them to that vault. 


On a PC you could just make a vault using the local OneDrive folder in apps if you want.

**Reply by james-s-w-clark** [reply]
> Make a new vault with the same name and then install the plugin to that vault. Configure the settings to connect to OneDrive and then it will sync them to that vault.
> 
> On a PC you could just make a vault using the local OneDrive folder in apps if you want.

Confirming this works, thanks!
I didn't know the vault's name had to match - that makes sense. 

---

On Windows, my "remotely-save" pushed my vault `myvault` to ` C:\Users\username\OneDrive\obsidian\myVaultName`

On Android, I created a vault `myvault`, set up remotely-save, and the files are all there.
It still works even if the Android vault is called `Myvault` (in case a user's Keyboard makes the first letter capitalised), good stuff! 👍
